Bottom: b216f34a27ab559e4171e7e53576be3bb2d06284
Top:    3d7ec5ba5995566bb563f476e49c56bfa262c142
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 16:06:40 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 60f063e..bc3f5d7 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -500,13 +500,3 @@ object3d_from_board_outline (void)
 
   return object;
 }
-
-void
-object3d_test_board_outline (void)
-{
-  object3d *board_outline;
-
-  board_outline = object3d_from_board_outline ();
-//  object3d_export_to_step (board_outline, "object3d_test.step");
-  destroy_object3d (board_outline);
-}
